DailyBudget/DailyBudgetWidget/BudgetEntityQuery.swift

```diff
@@ -0,0 +1,39 @@
+import AppIntents
+import SwiftData
+
+struct BudgetEntityQuery: EntityQuery {
+  func entities(for identifiers: [BudgetEntity.ID]) async throws -> [BudgetEntity] {
+    try await fetchEntities()
+      .filter { identifiers.contains($0.uuid) }
+      .map { BudgetEntity(budget: $0) }
+  }
+
+  /// Returns the initial results shown when a list of options backed by this query is presented.
+  func suggestedEntities() async throws -> [BudgetEntity] {
+    try await fetchEntities().map { BudgetEntity(budget: $0) }
+      .prefix(3)
```

Comment:
TODO: Test with more than 3 items in the store. I don't think it shows an option to select more, so we should make all active budgets available.